.gitignore

```diff
@@ -291,4 +291,5 @@ __pycache__/
 *.db
 
 # Visual Studio code
-.vscode/
\ No newline at end of file
+.vscode/
+/save-points/6-Deployment-docker/ConferencePlanner/conferenceplanner.yml
```

Comment:
Why is this added? If needed, can we make it more generic, e.g. /conferenceplanner.yml?